src/browserlib/extract-elements.mjs

```diff
@@ -216,7 +216,12 @@ export default function (spec) {
       // All elements defined in MathML Core
       // use the MathMLElement interface
       if (shortname === "mathml-core") {
-        elInfo.interface = "MathMLElement" ;
+        elInfo.interface = "MathMLElement";
+      }
+      // All elements of the Open Container Format of EPUB
+      // use the Element interface
+      else if (shortname === "epub-33") {
+        elInfo.interface = "Element";
```

Comment:
shouldn't we use null instead (and adapt the schema / tests accordingly)? if there is no expectation that an interface can be used for these, we shouldn't pretend there is.

Reply:
Possibly, I didn't know how to approach this... The Open Container Format is an XML dialect, so I would expect to get `Element` if I create a DOM out of a file that follows this format. I'm fine either way. I'll adjust the schema instead.